### PR TITLE
os/tools : Make binary header run under python3

### PR DESCRIPTION
**Pull request by YiHyunJin**
Python 2.7 will not be maintained past 2020.
So update python code to be able to run under python3.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [012c188459775cd0c8e5c7685df4c99e50f4efcd] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [012c188459775cd0c8e5c7685df4c99e50f4efcd] - Code Rule Check OK.

File: os/tools/mkbinheader.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 # language governing permissions and limitations under the License.
 #
 ############################################################################
+
+from __future__ import print_function
+from optparse import OptionParser
```
Comment (junmin-kim) [inline os/tools/mkbinheader.py]: Sorry for the confusion...
`from optparse import OptionParser` is not needed...

File: tools/ramdump/dumpParser.py [hunk 2]
```diff
@@ -73,14 +75,14 @@ def __init__(self, dump_file=None, elf=None, gdb_path=None, nm_path=None, readel
 		if self.elf is not None:
 			self.elf_file_fd = open(elf, 'rb')
 			if not self.elf_file_fd:
-				print('Failed to open {0}'.format(elf))
+				print(('Failed to open {0}'.format(elf)))
```
Comment (junmin-kim) [inline tools/ramdump/dumpParser.py]:
```suggestion
				print('Failed to open {0}'.format(elf))
```

**Comment by seinfra** [comment]
Target : [9a9b1e07b9b7968fd6d9675c8b126ffc929db11b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9a9b1e07b9b7968fd6d9675c8b126ffc929db11b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f3cdb9de573615fabf59a800faf57e568fc771cf] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f3cdb9de573615fabf59a800faf57e568fc771cf] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2e8b6a398b9275f5392f2840950a2576fef44a69] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2e8b6a398b9275f5392f2840950a2576fef44a69] - Code Rule Check (C++) OK.

**Pull request merged**